sites/bestbuy.py

```diff
@@ -6,327 +6,572 @@
     from Cryptodome.Cipher import PKCS1_OAEP
 from base64 import b64encode
 from utils import send_webhook, random_delay
-import requests, time, lxml.html, json, sys, settings
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.options import Options
+from selenium.common import exceptions
+from urllib import parse
+import requests, time, lxml.html, json, sys, settings, urllib3, threading, random
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
 
-# TODO: Implement settings.dont_buy global as to not checkout
 class BestBuy:
-    def __init__(self,task_id,status_signal,image_signal,product,profile,proxy,monitor_delay,error_delay):
-        self.status_signal,self.image_signal,self.product,self.profile,self.monitor_delay,self.error_delay = status_signal,image_signal,product,profile,float(monitor_delay),float(error_delay)
+
+    def __init__(self, task_id, status_signal, image_signal, product, profile, proxy, monitor_delay, error_delay):
+        self.status_signal, self.image_signal, self.product, self.profile, self.monitor_delay, self.error_delay = status_signal, image_signal, product, profile, float(monitor_delay), float(error_delay)
+        self.payment_id = None
         self.session = requests.Session()
+        self.sku
```

Comment:
this is raising `AttributeError: 'BestBuy' object has no attribute 'sku'`. Looks like it needs to be assigned

Reply:
I found this already during my other fixes. Already addressed it. It was from an earlier implementation approach.